### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Erkin97**
* update language files
* fix ssh key
* add error messages for key deployment
* minor changes in vb installation script

### CODE REVIEW

File: hub/src/RestWorker.cpp [hunk 1]
```diff
@@ -721,7 +721,24 @@ void CRestWorker::add_sshkey_to_environments(
...
+               http_status_code, err_code, network_error, doc_serialized, false);
+  qDebug() << "finished to add ssh keys"
+           << "json doc: " << doc
+           << "http code" << http_status_code
+           << "network code" << network_error
+           << "error code" << err_code;
+  if(err_code != RE_SUCCESS){
+    if(http_status_code == 500){
+      CNotificationObserver::Instance()->Error(tr("Failed to deploy ssh keys to the environments. "
+                                                  "Mostly it's because you already deployed a key with the same name or value"),
```
Comment (Erkin97) [inline hub/src/RestWorker.cpp]: @Almansherov look at this error message. when user tries to deploy key to the environment, but it has already key with the same key or value inside

File: hub/src/RestWorker.cpp [hunk 2]
```diff
@@ -721,7 +721,24 @@ void CRestWorker::add_sshkey_to_environments(
...
+                                               DlgNotification::N_NO_ACTION);
+    }
+    else{
+      CNotificationObserver::Instance()->Error(tr("Error occured during ssh key deployming to the environments."
+                                                  "Network error message: %1").arg(CCommons::NetworkErrorToString(network_error)),
+                                               DlgNotification::N_NO_ACTION);
```
Comment (Erkin97) [inline hub/src/RestWorker.cpp]: when some network errors occurs, when ssh key deployment

File: hub/src/RestWorker.cpp [hunk 3]
```diff
@@ -721,7 +721,24 @@ void CRestWorker::add_sshkey_to_environments(
...
+           << "json doc: " << doc
+           << "http code" << http_status_code
+           << "network code" << network_error
+           << "error code" << err_code;
+  if (err_code != RE_SUCCESS) {
+    if (http_status_code == 500) {
+      CNotificationObserver::Instance()->Error(tr("Failed to deploy ssh key to the environments. "
+                                                  "Mostly it's because some of the environments already has ssh key with "
+                                                  "the same name or value. Try to change name of ssh key or generate a new one."),
+                                               DlgNotification::N_NO_ACTION);
```
Comment (Erkin97) [inline hub/src/RestWorker.cpp]: new version of error message

File: hub/src/RestWorker.cpp [hunk 4]
```diff
@@ -721,7 +721,24 @@ void CRestWorker::add_sshkey_to_environments(
...
+    } else {
+      CNotificationObserver::Instance()->Error(tr("Error occured during deploying ssh key to the environments."
+                                                  "Network error message: %1").arg(CCommons::NetworkErrorToString(network_error)),
+                                               DlgNotification::N_NO_ACTION);
```
Comment (Erkin97) [inline hub/src/RestWorker.cpp]: here also new version

File: hub/src/RestWorker.cpp (hunk 1 above)
Comment (Almansherov) [reply hub/src/RestWorker.cpp]: "Failed to deploy SSH keys to the environments. Check the keys do not have the same name or value."

File: hub/src/RestWorker.cpp (hunk 2 above)
Comment (Almansherov) [reply hub/src/RestWorker.cpp]: "Unable to deploy SSH key to the environments."

File: hub/src/RestWorker.cpp (hunk 3 above)
Comment (Almansherov) [reply hub/src/RestWorker.cpp]: "Failed to deploy SSH keys to the environments. Check the keys do not have the same name or value and change them."

File: hub/src/RestWorker.cpp (hunk 4 above)
Comment (Almansherov) [reply hub/src/RestWorker.cpp]: "Unable to deploy SSH key to the environments."

**Pull request merged**